src/control/controlqml.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <QtQml>
+#include <memory>
+
+#include "control/controlproxy.h"
+
+class ControlQml : public QObject {
+    Q_OBJECT
+    Q_PROPERTY(QString group READ getGroup WRITE setGroup)
+    Q_PROPERTY(QString key READ getKey WRITE setKey)
+    Q_PROPERTY(double value READ getValue WRITE setValue NOTIFY valueChanged)
+    Q_PROPERTY(double parameter READ getParameter WRITE setParameter NOTIFY parameterChanged)
+
+  public:
+    ControlQml(QObject* parent = nullptr);
+
+    void setGroup(const QString& group);
+    const QString& getGroup() const;
+
+    void setKey(const QString& key);
+    const QString& getKey() const;
+
+    void setValue(double newValue);
+    double getValue() const;
+
+    void setParameter(double newValue);
+    double getParameter() const;
+
+  signals:
+    void valueChanged(double newValue);
+    void parameterChanged(double newParameter);
+
+  private slots:
+    void controlProxyValueChanged(double newValue);
+
+  private:
+    // QML cannot pass arguments to C++ constructors so this class
+    // needs to rely on the QML object setting the group and key
+    // properties to initialize the ControlProxy.
+    void initialize();
+
+    QString m_emptyString;
+    QString m_group;
+    QString m_key;
```

Comment:
```suggestion
    ConfigKey m_key;
```
This is how it is used in the other control objects. 
We have a conflict with group+item internaly and group+key externally. 
To fix this some more refactoring is required.

Reply:
>  We have a conflict with group+item internaly and group+key externally.

Yes, the terminology now is awkward. How can we fix this?

Reply:
We may renaim m_key internally to to m_coKey or something and then rename m_item to key.

For this PR it is sufficient to use ControlKey as suggested.